pywinauto/keyboard.py

```diff
@@ -76,6 +78,10 @@
 
 Repetition count can be specified for special keys. ``{ENTER 2}`` says to
 press Enter twice.
+
+To press and hold or release a key on the keyboard:
+``{VK_SHIFT down}pywinauto{VK_SHIFT up}`` #to type PYWINAUTO
+``{h down}{e down}{h up} {e up}llo``  # to type hello

```

Comment:
It's better to make a multi-line code section. Comments should be a part of code section. Also need a space after `#` symbol according to PEP8.

Reply:
@vasily-v-ryabov Is this what you meant? I tried to align this with the examples mentioned in the docstring in the section "Example how to use modifiers: "

```
    send_keys("{VK_SHIFT down}"
              "pywinauto"
              "{VK_SHIFT up}") # to type PYWINAUTO

    send_keys("{h down}"
              "{e down}"
              "{h up}"
              "{e up}"
              "llo") # to type hello
```